data_sinks/influxdb.go

```diff
@@ -51,6 +52,11 @@ func InfluxDB(conf config.InfluxDBPublisher) chan<- parser.Measurement {
 			if measurement.Name != nil {
 				p.AddTag("name", *measurement.Name)
 			}
+			if len(additionalTags) != 0 {
```

Comment:
This length check is unnecessary, a for range loop will skip over empty iterables, which also makes the additionalTags variable redundant

Reply:
Oh, good to know, thanks.